packages/navbar/src/navbar.tsx

```diff
@@ -24,10 +28,17 @@ export const UiNavbar: React.FC<UiNavbarProps> = ({
   category = 'primary',
   children,
   orientation = 'inline',
+  roundedCorners,
 }: UiNavbarProps) => {
   const themeContext = React.useContext(ThemeContext);
 
   const NavbarContent = React.useMemo(() => {
+    let numberOfItems = 0;
+
+    React.Children.map(children, () => {
+      numberOfItems++;
```

Comment:
Would get the same result by looking into the length of children?

Reply:
I tried to do so, but the `.length` was not available probably something with the typing? let me double check

Reply:
Maybe children.count https://react.dev/reference/react/Children#children-count

Reply:
I think this one will work nicely, let me check and update

Reply:
Yup, working nicely.l